Dockerfile.base

```diff
@@ -5,6 +5,6 @@ WORKDIR /root
 RUN dnf -y upgrade --setopt=install_weak_deps=False && \
     dnf -y install --setopt=install_weak_deps=False \
       bash bzip2 curl git make patch pkgconfig python3 scons unzip which xz && \
-    dnf clean all
+    dnf clean packages
```

Comment:
Don't delete the lists that just took 10min to download. Save them for all the other dockers rather than wasting that time over and over.

Reply:
Fair enough, but 10 min sounds really excessive to download DNF metadata. Do you have a particularly slow connection/mirror?

Reply:
I've been in 3 countries in asia in the last several months, and about 7 living spaces. It varies wildly, but I find that the fedora servers are very slow even if I have a good connection. Right now I have 10-20mbps, but dnf downloads at 20-80kbps.

Reply:
I've noticed that Internet connectivity from containers (at least ours) can be a bit iffy -- which also explains your stalls when cloning Mono, which I also experienced a few times. So that might be why.
Though it's true that even on bare metal, Fedora metadata updates is always an unnecessary slow step.